packages/front-end/src/app/components/EGLabView.vue

```diff
@@ -92,6 +95,8 @@
     };
   }
 
+  const isAllDataLoaded = computed(() => !useUiStore().anyRequestPending(['loadLabData', 'getSeqeraPipelines']));
```

Comment:
There are many loading requests on this page and different components relate to different sets of requests to be considered 'loading' or not.

The changes where `isAllDataLoaded` has replaced existing expressions are not quite right because of this. For example, if the Runs data has loaded, but the Pipelines data is still loading, the Pipelines tab should show loading but the Runs tab should not. This is how it was set up before.

I propose removing `isAllDataLoaded` and reverting the associated changes.

Reply:
Ah yep, sorry about that, I misread the request names and thought they were all repeated.